### PR TITLE
Allow expandable details for offline (incomplete) subscriptions

### PR DESCRIPTION
**Pull request by mohitb35**
1. /profile/recurrencies - Allows expansion of subscription details for `offline` payment method, even if `status='incomplete'
2. Shows transfer details for subscription with `offline` payment method,
3. Refactors `Recurrencies` to remove some repetitive code
4. Refactors `RecurrencyRecord` to adopt a similar structure to `AccountRecord`
5. Extracts `TransferDetails` from `AccountRecord` to facilitate reuse
6. Removes the Payments namespaces and imports dependencies instead.
7. Miscellaneous changes to resolve TS warnings, clean up code, remove unused imports and variables.
8. Removes "Pause" action for `offline` subscriptions (done after the screenshots below).

###### Update design
_Before_
Before - 1

_After_
After - 1

###### Show Transfer Details
_Before_
Before - 2

_After_
After - 2

Note: "Pause" action has been removed for `offline` subscriptions

###### Allow expandable details for offline (incomplete) subscriptions
_Before_
Before - 3

_After_
After - 3

### CODE REVIEW

**Comment by mariahosfeld** [comment]
Functionality works for me as expected.

**Review by mariahosfeld — COMMENTED** [review]
Functionality works as expected for me.

File: src/features/user/Account/Recurrency.tsx [hunk 1]
```diff
@@ -26,45 +21,38 @@ export default function Recurrency({
   recurrencies,
   fetchRecurrentDonations,
 }: Props): ReactElement {
-  const { t, i18n } = useTranslation(['me']);
+  const { t } = useTranslation(['me']);
   const [selectedRecord, setSelectedRecord] = React.useState<number | null>(0);
-  const [openModal, setOpenModal] = React.useState(false);
+  const [isModalOpen, setIsModalOpen] = React.useState(false);
+  // const [openModal, setOpenModal] = React.useState(false);
```
Comment (sunilsabatp) [inline src/features/user/Account/Recurrency.tsx:27]: if this state not needed anymore then remove it

**Comment by mohitb35** [comment]
@mariahosfeld 
Can offline subscriptions be paused and resumed? Currently, API allows pausing, but resume fails.

File: src/features/user/Account/Recurrency.tsx (hunk 1 above)
Comment (mohitb35) [reply src/features/user/Account/Recurrency.tsx]: Done

**Comment by mariahosfeld** [comment]
Functionality looks good to me.

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/profile/history**](https://planet-webapp-qlvqlqhu0-planetapp.vercel.app/profile/history) | [**/profile/recurrency**](https://planet-webapp-qlvqlqhu0-planetapp.vercel.app/profile/recurrency) |
|:-:|:-:|
|<a href="https://planet-webapp-qlvqlqhu0-planetapp.vercel.app/profile/history">Screenshot of /profile/history</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/00_apFv444HipW0EY2REKM0g1tVt7u2vgGicH3Nwtp3OF-dwOeIQSvBf-ascVtGezP0xyWEWESVSx6HhbRcwXw.png">(view full size)</a> |<a href="https://planet-webapp-qlvqlqhu0-planetapp.vercel.app/profile/recurrency">Screenshot of /profile/recurrency</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/b7hVY6utYCOJ6uDH0SjsoMNJhd9i5DVyEBiM2Ji50wlYSYqZNGI2jLNwrz1-yU54Ya_o1SboIz2j1QgpPFyW1g.png">(view full size)</a> |







Commit d36fe85bbce184df161a683de32cf3000b9886c8 (https://planet-webapp-qlvqlqhu0-planetapp.vercel.app).

**Comment by mariahosfeld** [comment]
Functionality still works correctly.

**Comment by mariahosfeld** [comment]
Functionality looks good.

**Review by sunilsabatp — APPROVED** [review]

**Pull request merged**